source/comply/legal-hold.rst

```diff
@@ -173,4 +173,19 @@ Legal Hold is an initial step to ensure relevant electronically stored informati
 How do I enable e-discovery for Mattermost?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Learn more about extracting data for e-discovery in our :doc:`product documentation </comply/electronic-discovery>`.
\ No newline at end of file
+Learn more about extracting data for e-discovery in our :doc:`product documentation </comply/electronic-discovery>`.
+
+How do I manage storage costs and version retention in S3?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``legalhold`` directory in your S3 bucket is subject to frequent modifications. This can result in the rapid accumulation of multiple object versions, significantly increasing storage costs and complicating version management over time. 
+
+To address these challenges, we strongly recommend configuring `S3 Lifecycle Rules <https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lifecycle-mgmt.html>`_ to efficiently manage storage, control costs, and ensure compliance with legal and organizational requirements.
+
+- Targeting only the ``legalhold`` directory ensures that other areas of your S3 bucket remain unaffected by these rules while minimizing the storage footprint and associated costs. Automating version control and cleanup reduces manual intervention, making bucket management more effective and scalable.
+- Review your organization's data retention policies to ensure compliance with legal and operational requirements. Choose appropriate timelines for transitioning and expiring file versions. 
```

Comment:
The `legalhold` directory is not part of the customer's retention policy. This line should be removed.